libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -157,7 +157,7 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @Param: SAFETY_DEFLT
     // @DisplayName: Sets default state of the safety switch
     // @Description: This controls the default state of the safety switch at startup. When set to 1 the safety switch will start in the safe state (flashing) at boot. When set to zero the safety switch will start in the unsafe state (solid) at startup. Note that if a safety switch is fitted the user can still control the safety state after startup using the switch. The safety state can also be controlled in software using a MAVLink message.
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:Enabled,3:Use Safety pin to control ProfiLED
```

Comment:
we still need BRD_SAFETY_DEFLT, most vehicles these days have safety state controlled via mavlink in software, not via a button

Reply:
Fixed by creating a separate parameter that enabled IO Profiled control. Also tested if Safety switch control continue to operate as before through mission planner.